src/utils/string/StringUtils.js

```diff
@@ -188,7 +189,6 @@ const getChatName = (usernames, me) => {
   if (userArraysWithoutMe.length === 1) {
     return userArraysWithoutMe[0].trim();
   }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getChatName` does not have a return statement if `userArraysWithoutMe.length` is not equal to 1. This could lead to unexpected behavior. Consider adding a default return value.
```diff
   if (userArraysWithoutMe.length === 1) {
     return userArraysWithoutMe[0].trim();
   }
+  return 'No Name';
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->